refractr/complex.py

```diff
@@ -74,7 +74,7 @@ def render_rewrite(self, match, target, status=None, location=False):
         rewrite = KeyMultiValueOption(
             'rewrite', [
                 match,
-                create_target(target),
+                create_target(target, self.preserve_path),
```

Comment:
i don't think adding this changes anything as the default is still True. I did compare Nginx created before & after for all the refracts with both this addition & without - the only difference is this new refract where we declare preserve-path